builtInFunctions/changeOwnerAddress.go

```diff
@@ -5,8 +5,8 @@ import (
 	"fmt"
 	"sync"
 
-	"github.com/ElrondNetwork/elrond-go-core/core/check"
-	vmcommon "github.com/ElrondNetwork/elrond-vm-common"
+	"github.com/multiversx/mx-chain-core-go/core/check"
+	vmcommon "github.com/multiversx/mx-chain-vm-common"
```

Comment:
can be replaced with "github.com/multiversx/mx-chain-vm-common" on the entire project